omnisafe/algorithms/off_policy/sac.py

```diff
@@ -0,0 +1,122 @@
+# Copyright 2022 OmniSafe Team. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""Implementation of the DDPG algorithm."""
+
+
+import torch
+
+from omnisafe.algorithms import registry
+from omnisafe.algorithms.off_policy.ddpg import DDPG
+
+@registry.register
+class SAC(DDPG):  # pylint: disable=too-many-instance-attributes
+    """Continuous control with deep reinforcement learning (DDPG) Algorithm.
+
+    References:
+        Paper Name: Continuous control with deep reinforcement learning.
+        Paper author: Timothy P. Lillicrap, Jonathan J. Hunt, Alexander Pritzel, Nicolas Heess,
+                      Tom Erez, Yuval Tassa, David Silver, Daan Wierstra.
+        Paper URL: https://arxiv.org/abs/1509.02971
+
+    """
+
+    def __init__(
+        self,
+        env_id: str,
+        cfgs=None,
+        algo: str = 'SAC',
+        wrapper_type: str = 'OffPolicyEnvWrapper',
+    ):
+        """Initialize DDPG."""
+        super().__init__(
+            env_id=env_id, 
+            cfgs=cfgs, 
+            algo=algo, 
+            wrapper_type=wrapper_type,
+            )
+        self.alpha = cfgs.alpha
+        self.alpha_gamma = cfgs.alpha_gamma
+
+    def compute_loss_v(self, data):
+        """
+        computing value loss
```

Comment:
different comment style. capital letter in line 42

Reply:
FIx it now. All comments will be capital.